Fixed typo

letsgitcracking opened the pull request:
There's a mismatch between sections in the source code and deployed website. There are only 4 sections on the site however in the repo there are 5 sections - this means when referencing sections its off by 1.

Perhaps changing 'Section 1' from this repo to 'Getting Started' might help.

adilanchian commented on Solidity_And_Smart_Contracts/Section_4/Lesson_2_Fund_Contract_Set_Prize.md:
```diff
@@ -56,10 +56,10 @@ is that it lets us make sure that the *balance of the contract* is bigger than t
 Pretty awesome, right :)?
 
-🏦 Fund the contract we can even send Ethereum!
+🏦 Fund the contract so can send Ethereum!
```
Little typo - can you change to this please:

`Fund the contract so we can send ETH`

adilanchian commented on Solidity_And_Smart_Contracts/Section_4/Lesson_2_Fund_Contract_Set_Prize.md:
```diff
@@ -56,10 +56,10 @@ is that it lets us make sure that the *balance of the contract* is bigger than t
 -----------------------------------------------
 
-We've now set up our code send Ethereum. Nice! Now we need to actually make sure our contract is funded, otherwise, we have no Ethereum to send!
+We've now set up our code to send Ethereum. Nice! Now we need to actually make sure our contract is funded, otherwise, we have no Ethereum to send!
```
Can you replace `Ethereum` here with `ETH` please?

adilanchian commented on Solidity_And_Smart_Contracts/Section_4/Lesson_2_Fund_Contract_Set_Prize.md:
````diff
@@ -117,7 +117,7 @@ const runMain = async () => {
 ```
 
-The magic is on `hre.ethers.utils.parseEther('0.001'),`. This where I say, "go and deploy my contract and fund it with 0.001 Ethereum". This will remove Ethereum from my wallet, and use it to fund the contract. **That's it**.
+The magic is on `hre.ethers.utils.parseEther('0.1'),`. This where I say, "go and deploy my contract and fund it with 0.1 Ethereum". This will remove Ethereum from my wallet, and use it to fund the contract. **That's it**.
````
What did you make this change?

letsgitcracking replied:
There's inconsistency in the code/instructions.

There is the actual prize amount which is 0.0001 ETH.

There is the part where you fund the contract with 0.1 ETH when you run locally i.e. scripts/run.js

There is the part where you deploy on Rinkeby with 0.001 ETH when you run scripts/deploy.js

I've had a look please review as soon as you can.

adilanchian reviewed (changes requested):
Had a couple asks and some questions!

Thanks so much :)

letsgitcracking commented:
I've changed all references to Ethereum when used in the context of a token to ETH. I've also updated the amount in the run.js part and the deploy.js part.

adilanchian commented:
POG this looks great! 

Thanks for the help here :)
